charts/zwave2mqtt/values.yaml

```diff
@@ -1,105 +1,57 @@
-# timezone: "America/New York"
-
-# device where the zwave device can be accessed
-device: "/dev/ttyASM0"
-
-imagePullSecrets: []
-nameOverride: ""
-fullnameOverride: ""
+strategy:
+  type: Recreate
 
-# Probes configuration
-probes:
-  liveness:
-    initialDelaySeconds: 30
-    failureThreshold: 5
-    timeoutSeconds: 10
-  readiness:
-    initialDelaySeconds: 30
-    failureThreshold: 5
-    timeoutSeconds: 10
-
-persistence:
-  enabled: false
-  ## zwave2mqtt configuration data Persistent Volume Storage Class
-  ## If defined, storageClassName: <storageClass>
-  ## If set to "-", storageClassName: "", which disables dynamic provisioning
-  ## If undefined (the default) or set to null, no storageClassName spec is
-  ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
-  ##   GKE, AWS & OpenStack)
-  ##
-  # storageClass: "-"
-  ##
-  ## If you want to reuse an existing claim, you can pass the name of the PVC using
-  ## the existingClaim variable
-  # existingClaim: your-claim
-  accessMode: ReadWriteOnce
-  size: 1Gi
-  ## Do not delete the pvc upon helm uninstall
-  skipuninstall: false
+env:
+  # TZ
+  OZW_AUTO_UPDATE_CONFIG: true
 
 service:
-  type: ClusterIP
-  port: 8091
-  ## Specify the nodePort value for the LoadBalancer and NodePort service types.
-  ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport
-  ##
-  # nodePort:
-  ## Provide any additional annotations which may be required. This can be used to
-  ## set the LoadBalancer service type to internal only.
-  ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
-  ##
-  annotations: {}
-  labels: {}
-  ## Use loadBalancerIP to request a specific static IP,
-  ## otherwise leave blank
-  ##
-  loadBalancerIP:
-  # loadBalancerSourceRanges: []
-  ## Set the externalTrafficPolicy in the Service to either Cluster or Local
-  # externalTrafficPolicy: Cluster
+  port:
+    port: 8091
 
-ingress:
-  enabled: false
-  annotations: {}
-    # kubernetes.io/ingress.class: nginx
-    # kubernetes.io/tls-acme: "true"
-  path: /
-  hosts:
-    - chart-example.local
-  tls: []
-  #  - secretName: chart-example-tls
-  #    hosts:
-  #      - chart-example.local
+securityContext:
+  privileged: true
 
-resources: {}
-  # We usually recommend not to specify default resources and to leave this as a conscious
-  # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
-
-nodeSelector: {}
-
-tolerations: []
-
-affinity: {}
-
-podAnnotations: {}
+persistence:
+  config:
+    enabled: false
+    emptyDir: false
+    mountPath: /usr/src/app/store
+    ## Persistent Volume Storage Class
```

Comment:
Do you want to keep these comments in? I don't think we have for the the other charts

Reply:
For now i think it's fine to add in. 

we're really lacking on documentation so once we get that covered in a centralized wiki or something a lot of charts will be updated to reference it instead

Reply:
Sounds like a plan! 👍